src/plugins/intel_npu/src/plugin/npuw/lazy_tensor.cpp

```diff
@@ -193,18 +211,10 @@ ov::Tensor LazyTensorImpl::eval() const {
             const auto& shape = std::get<3>(unpack_meta);
             const auto& type = std::get<4>(unpack_meta);
```

Comment:
it seems like  `variable1, variable2, ... = std::tie(unpack_meta)` will be useful here

Reply:
Done, thanks!